l10n_it_fatturapa_in/wizard/wizard_import_fatturapa.py

```diff
@@ -322,11 +322,12 @@ def _prepareInvoiceLine(
                     ('child_ids', '=', False),
                 ], context=context)
             if not account_tax_ids:
-                raise orm.except_orm(
-                    _('Error!'),
+                if context.get('inconsistencies'):
+                    context['inconsistencies'] += '\n'
+                context['inconsistencies'] += (
                     _('Define a tax with percentage '
```

Comment:
In questo caso scriverei "XML contains tax with percentage %s, but it does not exist in your system" perchè non è detto che debbano crearla